src/components/MultiHopTrade/components/TradeInput/components/RecipientAddress.tsx

```diff
@@ -157,7 +156,7 @@ export const RecipientAddress: React.FC<RecipientAddressProps> = ({
 
   const handleFormSubmit = useMemo(() => handleSubmit(onSubmit), [handleSubmit, onSubmit])
 
-  if (!receiveAddress || shouldForceManualAddressEntry || isAccountMetadataLoading) return null
+  if (!receiveAddress || shouldForceManualAddressEntry) return null
```

Comment:
This would produce a flash of recipient address as accounts are loading and this is useless - we don't need to know the loading status of accounts here, only whether or not we should force manual entry / whether or not we have a receive addy.

Reply:
From a logical point of view, I agree with your comment

But from a product point of view, it feels super weird to now display the recipient input because the destination chain is still loading 

![image](https://github.com/user-attachments/assets/dc395535-861a-4577-bc9c-8ce739612050)

I think we want to avoid this

Reply:
As discussed, we can't do this just yet, see https://github.com/shapeshift/web/pull/7896#discussion_r1787966893
We don't have a concept of queued accountIds for load just yet so for the time being, this is the best we can do as we will either have an ugly flash, ~2mn load with "Accounts Loading" (previous behaviour before the regression was introduced), or this

Reply:
Actually, getting the granular *account* loading state to work was more sane than trying to fix the toggle of hollistic account*s* loading state, see https://github.com/shapeshift/web/pull/7896#issuecomment-2399444219